scripts/imgtool/image.py

```diff
@@ -20,7 +20,15 @@
 Image signing and management.
```

Comment:
commit `35c9291fcafafe8608722e0ec3c801178884f0ef` does not exist

Reply:
Rebased on NCS and it seems that the additional commits, revert and bring new, are no longer needed, so all that is left is the commit with pure support.

Reply:
This cannot be applied until compression support is merged into upstream and your upstream PR is rebased and this cherry-picked again:
```
git cherry-pick e872745275fe6920494caf297b1ce996c59ea21f
Auto-merging scripts/imgtool/image.py
CONFLICT (content): Merge conflict in scripts/imgtool/image.py
Auto-merging scripts/imgtool/main.py
CONFLICT (content): Merge conflict in scripts/imgtool/main.py
error: could not apply e8727452... imgtool: Add pure signature support
hint: After resolving the conflicts, mark them with
hint: "git add/rm <pathspec>", then run
hint: "git cherry-pick --continue".
hint: You can instead skip this commit with "git cherry-pick --skip".
hint: To abort and get back to the state before "git cherry-pick",
hint: run "git cherry-pick --abort".
hint: Disable this message with "git config advice.mergeConflict false"
```
I will ping David to take a look today at the tech call later